src/Microsoft.IdentityModel.JsonWebTokens/JwtHeaderParameterNames.cs

```diff
@@ -28,69 +28,69 @@
 namespace Microsoft.IdentityModel.JsonWebTokens
 {
     /// <summary>
-    /// List of header parameter names see: http://tools.ietf.org/html/rfc7519#section-5.
+    /// List of header parameter names see: https://datatracker.ietf.org/doc/html/rfc7519#section-5.
     /// </summary>
     public struct JwtHeaderParameterNames
     {
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7515#section-4.1.1
+        /// See: https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.1
         /// </summary>
         public const string Alg = "alg";
 
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7515#section-4.1.10
-        /// also:https://tools.ietf.org/html/rfc7519#section-5.2
+        /// See: https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.10
+        /// also:https://datatracker.ietf.org/doc/html/rfc7519#section-5.2
```

Comment:
```suggestion
        /// Also: https://datatracker.ietf.org/doc/html/rfc7519#section-5.2
```

Reply:
changed in e89d4fbafb9ac32deb2b8b8a704e473b9e647669